src/modules/rc_update/params.c

```diff
@@ -1924,6 +1956,23 @@ PARAM_DEFINE_INT32(RC_MAP_ENG_MOT, 0);
  */
 PARAM_DEFINE_INT32(RC_FAILS_THR, 0);
 
+/**
+ * Threshold for selecting R/C enable switch
+ *
+ * 0-1 indicate where in the full channel range the threshold sits
+ * 		0 : min
+ * 		1 : max
+ * sign indicates polarity of comparison
+ * 		positive : true when channel>th
+ * 		negative : true when channel<th
+ *
+ * @min -1
+ * @max 1
+ * @decimal 2
+ * @group Radio Switches
```

Comment:
I appreciate this is copied from another parameter, but I think it is is bit confusing.

```suggestion
 * Threshold for selecting R/C enable switch
 *
 * The absolute value indicates the position of the threshold in the channel range,
 * between 0 and 1.
 * The sign (+ or -) indicates whether the switch is active when 
 * the channel value is above (positive) or below (negative) the threshold:
 * For example, 0.75 indicates the switch is active when its value is more
 * than 0.75 of the total range.
 *
 * @min -1
 * @max 1
 * @decimal 2
 * @group Radio Switches
```